tests/stress/stress_test.go

```diff
@@ -97,9 +97,10 @@ func stopNodes(cancel context.CancelFunc, runtimeErrors []<-chan error) {
 	}
 }
 
+// TODO: add tests against latest dev runtime
```

Comment:
On that note, when should we do this? Is it the next upcoming PR?
Also should we keep older tests testing against the older runtime? 🤔

Reply:
as mentioned in standup. The issues are created, and we should be tackling them as soon as we are finished with the current initiatives.

Reply:
okay just saw this, addressed my above questions